package.json

```diff
@@ -20,9 +22,30 @@
   "bugs": {
     "url": "https://github.com/appium/appium-uiautomator/issues"
   },
+  "engines": [
+    "node"
+  ],
   "homepage": "https://github.com/appium/appium-uiautomator",
   "dependencies": {
-    "underscore": "~1.6.0",
-    "winston": "~0.7.2"
+    "adm-zip": "~0.4.4",
+    "appium-adb": "^2.0.0-beta4",
+    "appium-logger": "^1.0.0",
+    "babel-runtime": "=5.5.5",
+    "jsonwp-status": "0.0.1",
+    "lodash": "^3.9.1",
+    "q": "~1.0.1",
+    "rimraf": "~2.2.8",
+    "teen_process": "^1.2.2",
+    "winston": "~0.7.3"
```

Comment:
thanks for pointing that out will revisit dependencies, i deleted bunch of code from bootstrap